### ISSUE TITLE
Reproducible build failed for v1.5

### ISSUE THREAD

**Issue by IzzySoft** [catpuppyapp/SshKeyMan#3]
While 1.3 was verified fine, today v1.5 failed verification. APK diff shows `classes.dex` does not match:

```diff
  -rw-r--r--  0.0 unx      120 b-      117 defN 1981-01-01 01:01:02 ba7572b3 META-INF/version-control-info.textproto
- -rw-r--r--  0.0 unx     4784 b-     4784 stor 1981-01-01 01:01:02 e2aba3b1 assets/dexopt/baseline.prof
+ -rw-r--r--  0.0 unx     4787 b-     4787 stor 1981-01-01 01:01:02 70c079eb assets/dexopt/baseline.prof
  -rw-r--r--  0.0 unx      599 b-      599 stor 1981-01-01 01:01:02 30df0875 assets/dexopt/baseline.profm
- -rw-r--r--  0.0 unx  4450172 b-  2029796 defN 1981-01-01 01:01:02 bfb6d9fc classes.dex
+ -rw-r--r--  0.0 unx  4449760 b-  2029435 defN 1981-01-01 01:01:02 6bcce594 classes.dex
  -rw-r--r--  0.0 unx    10096 b-    10096 stor 1981-01-01 01:01:02 9734baa0 lib/arm64-v8a/libandroidx.graphics.path.so
```

(the diff in `baseline.prof` is just a result in that, as the checksum of `classes.dex` is saved there). The diff in `classes.dex` is rather large (~240 kB).

Was that APK really built from a clean tree at the commit the tag points to, or did some leftovers (artifacts from previous builds) sneak in – or did you apply local chances after that commit but before the APK was built? Or did something else change (e.g. the JDK version: which one are you currently using?)?

**Comment by Bandeapart1964** [comment]
The release is build from the tag pointed commit, idk whats wrong, 

and the jdk version... my pc installed jdk21/jdk11/jdk8, and Android Studio bundled a jdk too, so i don't really know which are using by Android Studio and gradle.

this release I was edited the ProGuard file, idk maybe it has some effect for the baseline or classes.dex,

next time, before I build, I'll try gradle clean first, maybe will sovle this issue.

**Comment by Bandeapart1964** [comment]
hi try new version, maybe resloved,I using Android Studio build it, and before build I do a ./gradlew clean, hope it can pass the reporducible:

https://github.com/catpuppyapp/SshKeyMan/releases/tag/1.6v7

**Comment by IzzySoft** [comment]
Whatever it was, your new release solved it :partying_face: 1.6v7 was just confirmed RB, so with the next sync the green shield will be up again. Thanks a lot!

> next time, before I build, I'll try gradle clean first, maybe will sovle this issue.

Yes, definitely! Always build from a clean tree (without artifacts from previous builds etc).
